app/grandchallenge/evaluation/models.py

```diff
@@ -1048,6 +1048,10 @@ def get_absolute_url(self):
     def get_peer_images(self):
         return Method.objects.filter(phase=self.phase)
 
+    @property
+    def linked_file(self):
+        raise self.image
```

Comment:
This only needs to be set on the abstract `ComponentImage` as it is the same for `AlgorithmImage`s, `WorkstationImage`s and `Method`s, right?

Does the deletion work when using the bulk delete in the admin?

Reply:
Yes, this works with the bulk delete in the admin. And you're right, the property can be set on the abstract class